acquisitions/src/main/resources/thunderjet/mod-orders/features/create-open-composite-order.feature

```diff
@@ -0,0 +1,157 @@
+      * print '## Prepare finances'
+      * configure headers = headersAdmin
+      * def histFundId = call uuid
+      * def genrlFundId = call uuid
+      * def miscHistFundId = call uuid
+      * def histBudgetId = call uuid
+      * def genrlBudgetId = call uuid
+      * def miscHistBudgetId = call uuid
+      * def v = call createFund { id: "#(histFundId)" }
+      * def v = call createBudget { id: "#(histBudgetId)", fundId: "#(histFundId)", allocated: 1000 }
+      * def v = call createFund { id: "#(genrlFundId)" }
+      * def v = call createBudget { id: "#(genrlBudgetId)", fundId: "#(genrlFundId)", allocated: 1000 }
+      * def v = call createFund { id: "#(miscHistFundId)" }
+      * def v = call createBudget { id: "#(miscHistBudgetId)", fundId: "#(miscHistFundId)", allocated: 1000 }
+      * configure headers = headersUser
+
+      * print '## Prepare the order'
+      * def po = read('classpath:samples/mod-orders/compositeOrders/po-listed-print-monograph.json')
+      * def line1 = po.compositePoLines[0]
+      * def line2 = po.compositePoLines[1]
+      * set line1.fundDistribution[0].fundId = histFundId
+      * set line1.fundDistribution[1].fundId = genrlFundId
+      * set line2.fundDistribution[0].fundId = miscHistFundId
+      * set line1.paymentStatus = "Pending"
+      * set line1.receiptStatus = "Partially Received"
+      * set line2.paymentStatus = "Pending"
+      * set line2.receiptStatus = "Partially Received"
+
+      # remove productId from PO line to test scenario when it's not provided so there is no check for existing instance but new one will be created
+      * remove line1.details.productIds[0]
+
+      # MODORDERS-117 only physical quantity will be used
+      * set line1.orderFormat = "Physical Resource"
+      * remove line1.eresource
+
+      # Set locations quantities
+      * set line1.locations[1].quantityPhysical = 3
+      * set line1.locations[2].quantityPhysical = 7
+      * set line1.locations[2].quantityElectronic = 0
+
+      # Set cost quantities
+      * set line1.cost.quantityPhysical = 11
+      * set line1.cost.quantityElectronic = 0
+      * set line1.cost.listUnitPrice = 10.0
+      * set line1.cost.listUnitPriceElectronic = 0.0
+
+      # Set status to Open
+      * set po.workflowStatus = "Open"
+
+      * print '## Create the composite order'
+      Given path 'orders/composite-orders'
+      And request po
+      When method POST
+      Then status 201
+
+      * print '## Check the returned composite order'
+      # Order
+      * def getCurrentDateUTC = function() { return java.time.LocalDateTime.now(java.time.ZoneOffset.UTC).format(java.time.format.DateTimeFormatter.ofPattern("yyyy-MM-dd")) }
+      * match $.dateOrdered contains getCurrentDateUTC()
+      * match $.id == '#notnull'
+      * def orderId = $.id
+      * match $.poNumber == '#notnull'
+      * match $.compositePoLines == '#[2]'
+      * match $.workflowStatus == "Open"
+      * match $.totalItems == 14
```

Comment:
line 88 use 'response' instead of '$'. Can we use same either '$' or 'response' that makes easier to follow flow

Reply:
We can't use $ everywhere, and response is a bit verbose. I could redefine the po variable to make the code more consistent.